src/pudl/metadata/templates/metadata.yml.jinja

```diff
@@ -46,33 +49,40 @@ databases:
       <a href="https://github.com/catalyst-cooperative/pudl/issues">make an issue
       on GitHub</a> to let us know.</li>
       <li>You can reach the PUDL team at
-      <a href="mailto:pudl@catalyst.coop">pudl@catalyst.coop</a>.</li>
+      <a href="mailto:{{ datasources['pudl'].email }}">{{ datasources['pudl'].email }}</a>.</li>
       <li>If you find our work useful, please consider
       <a href="https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=PZBZDFNKBJW5E&source=url">making
       a recurring contribution</a> to support it.</li>
       </ul></p>
     tables:
-      plants_entity_eia:
-        label_column: plant_name_eia
-      plants_ferc1:
-        label_column: plant_name_ferc1
-      plants_pudl:
-        label_column: plant_name_pudl
-      utilities_entity_eia:
-        label_column: utility_name_eia
-      utilities_ferc1:
-        label_column: utility_name_ferc1
-      utilities_pudl:
-        label_column: utility_name_pudl
+    {%- for resource in package.resources %}
```

Comment:
I pruned out the EPA CEMS resources by filtering the resources in the `Package` by `field_namespace`. I used `field_namespace` instead of providing a list of data sources because the source for the `glue` fields is `eia_epa_crosswalk` so it was a little simpler to use `field_namespace`. I'm not sure if `field_namespace` is really the best way to do it.

Reply:
The ETL selects data sources like `eia923`, `eia860` and `ferc1` for inclusion in the database that's output. each of them is composed of a collection of tables (resources). Which resources are associated with each data source is determined by the `etl_group` attribute in the resource definitions.  The `field_namespace` is a higher level category, where `eia` also includes `eia861` even though it's not currently being loaded into the database.

Would it make sense to do something like this?

```py
data_sources = ["eia860", "eia923", "ferc1", "pudl", "glue"]
resource_ids = []
for source in data_sources:
    resource_ids = resource_ids + DataSource.get_resource_ids(source)
package = Package.from_resource_ids(resource_ids)
```